src/Core/Time/Chronometer.cpp

```diff
@@ -10,6 +10,11 @@ namespace obe::Time
         m_started = true;
     }
 
+    bool Chronometer::started() const {
+        if(m_started) 
```

Comment:
```cpp
return m_started;
```
does the same thing but is more simple :)

Reply:
Alright 👍🏾